src/services/refactors/extractMethod.ts

```diff
@@ -934,12 +934,8 @@ namespace ts.refactor.extractMethod {
      * Otherwise, return `undefined`.
      */
     function getNodeToInsertBefore(minPos: number, scope: Scope): Node | undefined {
-        const children = getStatementsOrClassElements(scope);
-        for (const child of children) {
-            if (child.pos >= minPos && isFunctionLike(child) && !isConstructorDeclaration(child)) {
-                return child;
-            }
-        }
+        return find(getStatementsOrClassElements(scope), child =>

```

Comment:
Since this is a simple find loop, I think using a helper makes sense.  In general though, in the absence of expression-level breakpoints (and I would be very happy to learn that these exist), loops are much easier to debug.

Reply:
I think node.js recently started supporting column breakpoints -- should be in chrome too.